app/controllers/users_controller.rb

```diff
@@ -1,31 +1,29 @@
 class UsersController < ApplicationController
-  before_action :set_user, only: %i[ show edit update destroy ]
+  before_action :set_user, only: %i[show edit update destroy]
 
   # GET /users or /users.json
   def index
     @users = User.all
   end
 
   # GET /users/1 or /users/1.json
-  def show
-  end
+  def show; end
 
   # GET /users/new
   def new
     @user = User.new
   end
 
   # GET /users/1/edit
-  def edit
-  end
+  def edit; end
```

Comment:
I think we are not gonna need this actions, if you agree you could delete them before merging